### ISSUE TITLE
Firefox: Accessibility labels and event type labels on event preview cards not readble by a screen reader

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6002]
**What**
On the events preview cards on the "whats on pages" the mac voice over, screen reader cant read the little labels that are indicating that it is "speech to text" etc

**Where**

https://wellcomecollection.org/#main

<img width="1118" alt="Screenshot 2021-01-21 at 15 52 29" src="https://user-images.githubusercontent.com/7315126/105375553-b583c800-5c00-11eb-9de5-bc15f6128790.png">

**Comment by gestchild** [comment]
It seems that voiceover with firefox is ignoring any list items inside the anchor - not sure what to do about this as they are lists, so should be marked up as such
